### PR TITLE
Review for JS tasks and YouTube API task 

### PR DESCRIPTION
**Pull request by rajkumar-telu**
Hello Garv, 
Raising Pull Request after making the changes as you have mentioned.
Kindly give your rating in learn.epam.com too in your free time.
Thank You

### CODE REVIEW

File: Basics of JS/2.2/Get_The_Middle_Character.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function getMiddle(s)
+{
+  if(s.length%2){
```
Comment (sabharwal-garv) [inline Basics of JS/2.2/Get_The_Middle_Character.js]: you can assign length to a variable

File: Basics of JS/2.3/Closures_And_Scopes.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function createFunctions(n){
+    var callbacks = [];
+
+    for(let i=0;i<n;i++){
+        callbacks.push(function(){
+            return i;
+        });
+    }
```
Comment (sabharwal-garv) [inline Basics of JS/2.3/Closures_And_Scopes.js:8]: return callback array

Comment (rajkumar-telu) [reply Basics of JS/2.3/Closures_And_Scopes.js:8]: sorry Garv must have missed by mistake. Have corrected it in the [e8bffe1](https://github.com/rajkumar-telu/HomeTasks-Epam/pull/2/commits/e8bffe13f545117b31ecf4e796afa27566ee6899) commit.

File: Basics of JS/2.2/Get_The_Middle_Character.js (hunk 1 above)
Comment (rajkumar-telu) [reply Basics of JS/2.2/Get_The_Middle_Character.js]: Ok garv, I have assigned the length to the stringLength variable in the [e8bffe1](https://github.com/rajkumar-telu/HomeTasks-Epam/pull/2/commits/e8bffe13f545117b31ecf4e796afa27566ee6899) commit.

**Pull request merged**